libs/platform/user/ebpf_platform_user.cpp

```diff
@@ -510,7 +512,7 @@ ebpf_is_non_preemptible_work_item_supported()
 bool
 ebpf_is_preemptible_work_item_supported()
 {
-    return false;
+    return true;
 }
```

Comment:
Can't this function be deleted now?  As far as I can find, the only caller was the ebpf_native.c call that you deleted in this PR.  Same in ebpf_platform_kernel.c

Reply:
Removed.